src/style.css

```diff
@@ -0,0 +1,113 @@
+/* html {
```

Comment:
don't leave comments

Reply:
is not fixed for the full page. Remove` /* font-family: inherit; */` comment